wfe/wfe.go

```diff
@@ -537,6 +543,7 @@ func link(url, relation string) string {
 func (wfe *WebFrontEndImpl) NewRegistration(ctx context.Context, logEvent *requestEvent, response http.ResponseWriter, request *http.Request) {
 
 	body, key, _, prob := wfe.verifyPOST(ctx, logEvent, request, false, core.ResourceNewReg)
+	addRequesterHeader(response, logEvent.Requester)
```

Comment:
I put it before `prob != nil` because there are a handful of errors that could be set in `verifyPOST` where `logEvent.Requester` would be set. 
If `addRequesterHeader` is after `prob != nil` responses would not contain the Requester header for responses that could potentially have the header set. If this is the desired functionality (or is a security issue) then I will make the changes as requested.